setup.py

```diff
@@ -45,7 +45,7 @@
     include_package_data=True,
     python_requires=">=3.6",
     install_requires=[
-        'Django>=3.1,<3.2',
+        'Django>=3.2,<4.1',
```

Comment:
Does it work with Django 4.0?

Reply:
Yes, actually that is the only thing I have been testing it with and 3.2